README.md

```diff
@@ -1,24 +1,149 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
- [ ] These links in your README do not have sections for them. Kindly remove them or add their sections. We should avoid having broken links in the README.

  ![image](https://github.com/Progress2002/hello-rails-back-end/assets/36542986/93fb5f34-89f1-438f-9583-85c5075f86ea)

- [ ] You have the same issue on the frontend repo as well. Kindly fix that one too.

Reply:
Hello @TedLivist, thanks for the review and code suggestions, the project has been updated.